Lil-Peep/src/styles/modules/featured-kits.scss

```diff
@@ -0,0 +1,387 @@
+.featured-kits {
+  font-family: 'Oswald', sans-serif;
+
+  .big-banner {
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+}
+
+.big-banner-content {
+  background: linear-gradient(0deg,
+    rgb(251, 239, 227),
+    rgba(251, 239, 227, 1) 20%,
+    rgba(251, 239, 227, 0.2) 50%,
+    rgba(251, 239, 227, 0.1));
+  display: flex;
+  flex-direction: column;
+  height: 320px;
+  justify-content: flex-end;
+  text-align: center;
+
+  .big-banner-title {
+    font-size: 20px;
+    margin-bottom: 0;
+  }
+
+  .big-banner-description {
+    font-family: 'Lato', sans-serif;
+    font-size: 12px;
+    margin: 0;
+  }
+
+  .view-in-detail {
+    background-color: #4a90e2;
+    border-radius: 3px;;
+    border-style: none;
+    margin: 10px;
+    padding: 10px 0 10px 0;
+    text-align: center;
+
+    a {
+      text-decoration: none;
+
+      &:hover {
+        cursor: pointer;
+        background-color: #ed7f03;
+      }
+    }
+
+    .view-detail-button {
+      color: #fff;
+      font-family: 'Oswald', sans-serif;
+      font-weight: bold;
+      margin: 0;
+    }
+  }
+}
+
+.small-banner {
+  align-items: center;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  box-shadow: inset 0 0 0 1000px rgba(245, 166, 35, 0.8);
+  color: #FFFFFF;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 20px 20px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `20px` instead of `20px 20px`